package.json

```diff
@@ -39,16 +40,16 @@
     "@commitlint/cli": "^18.6.0",
     "@commitlint/config-conventional": "^18.6.0",
     "@commitlint/cz-commitlint": "^18.6.0",
-    "@storybook/addon-a11y": "^7.4.1",
-    "@storybook/addon-essentials": "^7.4.1",
-    "@storybook/addon-interactions": "^7.6.17",
-    "@storybook/addon-links": "^7.4.1",
+    "@storybook/addon-a11y": "7.6.20",
```

Comment:
Can I get a tldr of why we need to this upgrade? Is it because the design plugin requires it?

Reply:
nope this change was due to first commit and all changes in that commit were made via

```bash
npx storybook@7 upgrade
npm dedup
```

the first command is from the [storybook docs](https://storybook.js.org/docs/configure/upgrading) for upgrading and the second command was recommended by the CLI of the first to resolve duplication

Reply:
I'm sure it's harmless and glad you followed SB docs for _how_ to upgrade, but still wondering _why_ the upgrade. If it was just to get on the most recent 7.x version for the sake of being up to date, that's fine, just want to know because I've come across breaking changes sneaking into minor SB releases before.